src/pybind/mgr/crash/module.py

```diff
@@ -114,6 +114,16 @@ def time_from_string(self, timestr):
         except ValueError:
             return datetime.datetime.strptime(timestr, OLD_DATEFMT)
 
+    def user_friendly_timestamp(self, timestr):
+        # Converts timestamp as stored in crash metadata into a more
+        # user-friendly format. See https://tracker.ceph.com/issues/43897
```

Comment:
nit: I don't see much value in this comment here, especially in adding the tracker url. The purpose of the function is self evident.